DesignSafe-CI/portal
# DES-1897: changes to pub preview for all proj typs

## Overview: ##
Remove "Last Modified" column from "Publication Preview" file listing

## PR Status: ##


## Related Jira tickets: ##

* [DES-1897](https://jira.tacc.utexas.edu/browse/DES-1897)

## Summary of Changes: ##
added 'published' to FilesListingComponent
added conditional to check if 'Last Modified' column should be displayed
added published=true to all project types file listings

## Testing Steps: ##
1. Checked Working Directory and Curation Directory to make sure "Last Modified" IS still there. 
2. Checked Published Preview to make sure "Last Modified" IS NOT there.

## UI Photos:

## Notes: ##
